### ISSUE TITLE
Unable to initialize 64-bit visual studio build environment

### ISSUE THREAD

**Issue by arifsolmaz35** [cboulay/PSMoveService#552]
hello PSMoveSteamVRBridge,
I have Alpha Release # 8.10.0 installed.
I pressed InitialSetup_X64

Could Not Find C: \ Program Files \ PSMoveService \ PSMoveService-0.9-alpha8.10.0 \ SetBuildVars_x64.bat
The system can not find the path specified.
"Unable to initialize 64-bit visual studio build environment"
Press any key to continue. . .

**Comment by arifsolmaz35** [comment]
help me please

**Comment by cboulay** [comment]
You probably don't want to download and build PSMoveService from a C:\Program Files\ directory. You should run it from a user directory where you have write permissions.

**Comment by krzysztofcyran93** [comment]
yeah I'm doing it from different partition than C and it still shows me the same error...
